packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.test.ts

```diff
@@ -462,7 +462,10 @@ describe("movePoint on an angle graph", () => {
         invariant(updated.type === "angle");
 
         expect(
-            findAngle(updated.coords[0], updated.coords[2], updated.coords[1]),
+            getClockwiseAngle(
+                updated.coords,
+                updated.allowReflexAngles || false,
```

Comment:
Comment: Can possibly be shortened. Apologies if you can't actually shorten this and I just keep pointing out out 😅

Reply:
Thank you for finding these!! I only updated the content editor function, and missed doing it on the getClockwiseAngle method itself. I've updated them all now, and it's much cleaner!